### ISSUE TITLE
Default user id generation to auto inc

### ISSUE THREAD

**Issue by jpiepkow** [jpiepkow/accesspass#26]
Currently user generation is a random short binary. It was made this way originally because this project was first closed source. AccessPass should default to the standard auto inc because it already allows override of id generation. For this to work https://github.com/jpiepkow/accesspass/issues/7 must be done first.

**Comment by dfalling** [comment]
What are your thoughts on also using an `id` field instead of `user.user_id`?  It's the standard for any autogen in Phoenix.  Currently I'm changing it in my view and tests, but it's definitely inconsistent with my other data.

**Comment by jpiepkow** [comment]
I agree. This should be handled when allowing for custom schemes(is another issue). I have no timeline when I will be able to get to that issue currently though. Because I started down the path of user_id I would rather allow a custom override as compared to changing the default.

**Comment by jpiepkow** [comment]
If you have any solutions you want to try feel free to pull request
